src/mail.sh

```diff
@@ -1207,6 +1304,10 @@ function parse_mail_options()
         options_values['GROUPS_CREATE']=1
         shift
         ;;
+      --remove)
+        options_values['GROUPS_REMOVE']=1
+        shift
```

Comment:
Again, document these features on the man page. Also, these names are too generic. You should use something like remove-group.